submissions/xT0PERx/js-dom/code.js

```diff
@@ -1,65 +1,71 @@
+      '<img class="img-content" src=https://i1.wp.com/autogeek.com.ua/wp-content/uploads/2017/01/3f39410d0e6da7cacc8b6f4260bfd6d4-387x480.jpg?resize=387%2C480><p> Tesla (ранее Tesla Motors) — американская компания, производитель электромобилей и (через свой филиал SolarCity) решений для хранения электрической энергии.</p><p>Tesla разворачивает сеть «Суперзарядок» (англ. Supercharger) — станций для зарядки электромобилей, разработанных для того, чтобы на автомобилях Tesla можно было совершать длительные поездки. </p>',
   },
 ];
 
-document.querySelector(".menu-list").addEventListener("click", function (e) {
-  if (e.target && e.target.nodeName === "LI") {
+document.querySelector('.menu-list').addEventListener('click', function (e) {
+  if (e.target && e.target.nodeName === 'LI') {
     switch (e.target.className) {
-      case "menu-item-1":
-        showContent("menu-item-1");
+      case 'menu-item-1':
+        showContent('menu-item-1');
         break;
-      case "menu-item-2":
-        showContent("menu-item-2");
+      case 'menu-item-2':
+        showContent('menu-item-2');
         break;
-      case "menu-item-3":
-        showContent("menu-item-3");
+      case 'menu-item-3':
+        showContent('menu-item-3');
         break;
-      case "menu-item-4":
-        showContent("menu-item-4");
+      case 'menu-item-4':
+        showContent('menu-item-4');
         break;
-      case "menu-item-5":
-        showContent("menu-item-5");
+      case 'menu-item-5':
+        showContent('menu-item-5');
         break;
     }
   }
 });
 
 function showContent(nameClass) {
-  const element = document.querySelector("#content");
+  const element = document.querySelector('#content');
```

Comment:
I think, you can move this variable to a higher scope. Avoid of creating new variable every time you call function.
Move out this from `showContent` to global scope.